doc/input-output-reference/src/overview/group-internal-gains-people-lights-other.tex

```diff
@@ -2348,6 +2348,168 @@ \subsection{Internal Gain Equipment Outputs}\label{outputs-5-004}
+
+An IDF example:
+
+\begin{lstlisting}
+IndoorLivingWall,
+  Space1-1IndoorLivingWall,   !-Name
+  SPACE1-1SouthPartition,     !-Surface Name
+  AlwaysOn,                   !-Schedule Name
+  Penman-Monteith,            !-ET Calculation Method
+  LED,                        !-Lighting Method
+  AlwaysOff,                  !-LED Intensity Schedule Name
+  ,                           !-Daylighting Control Name
+  ,                           !-LED - Daylight Targeted Lighting Intensity Schedule Name
+  30,                         !- Total Leaf Area {m2}
+  32.5,                       !- LED Nominal Intensity {umol/(m2s)}
+  640,                        !- LED Nominal Power {W}
+  0.6;                        !- Radiant Fraction of LED Lights
+\end{lstlisting}
+
+\subsubsection{Outputs}\label{outputs-indoorlivingwall}
+
+IndoorLivingWall objects have output variables for individual objects. The following outputs are available:
+
+\begin{itemize}
+\tightlist
+\item
+  Zone,Average,Indoor Living Wall Plant Surface Temperature {[}C{]}
+\item
+  Zone,Average,Indoor Living Wall Sensible Heat Gain Rate {[}W{]}
+\item
+  Zone,Average,Indoor Living Wall Latent Heat Gain Rate {[}W{]}
+\item
+  Zone,Average,Indoor Living Wall Evapotranspiration Rate {[}\unit{\kilo\gram\per\square\meter\per\second}{]}
+\item
+  Zone,Average,Indoor Living Wall Energy Required For Evapotranspiration Per Unit Area {[}\unit{W/m^2}{]}
+\item
+  Zone,Average,Indoor Living Wall LED Operational PPFD {[}\unit{\micro\mole\per\square\meter\per\second}{]}
+\item
+  Zone,Average,Indoor Living Wall PPFD {[}\unit{\micro\mole\per\square\meter\per\second}{]}
+\item
+  Zone,Average,Indoor Living Wall Vapor Pressure Deficit {[}Pa{]}
+\item
+  Zone,Average,Indoor Living Wall LED Sensible Heat Gain Rate {[}W{]}
+\item
+  Zone,Average,Indoor Living Wall LED Operational Power {[}W{]}
+\item
+  Zone,Sum,Indoor Living Wall LED Electricity Energy {[}J{]}
+\end{itemize}
+
+\paragraph{Baseboard Electricity Rate {[}W{]}}\label{baseboard-electric-power-w}
+
+This field is the electric power for the ZoneBaseboard:OutdoorTemperatureControlled object in Watts.
+
+\paragraph{Baseboard Electricity Energy {[}J{]}}\label{baseboard-electric-energy-j}
+
+The outdoor temperature controlled baseboard heat option is assumed to be fueled by electricity. This field is the same as the Baseboard Total Heating Energy (above) in joules. This energy is included in the following meters:
+\begin{lstlisting}
+Electricity:Facility
+Electricity:Building
+Electricity:Zone:<Zone Name>
+Electricity:SpaceType:<Space Type Name>
+InteriorEquipment:Electricity
+InteriorEquipment:Electricity:Zone:<Zone Name>
+InteriorEquipment:Electricity:SpaceType:<Space Type Name>
+<End-Use Subcategory>:InteriorEquipment:Electricity
+<End-Use Subcategory>:InteriorEquipment:Electricity:Zone:<Zone Name>
+<End-Use Subcategory>:InteriorEquipment:Electricity:SpaceType:<Space Type Name>
+\end{lstlisting}
```

Comment:
Copy pasting

Reply:
Thank you very much for catching this! This section should be removed. @Myoldmopar, I will update this by removing this redundant section from the input-output manual and submit a request.